tez-api/src/main/java/org/apache/tez/runtime/api/InputInitializerContext.java

```diff
@@ -87,7 +87,12 @@ public interface InputInitializerContext {
    * @return Resource
    */
   Resource getVertexTaskResource();
-  
+
+  /**
+   * Get the vertex id as integer that belongs to this input.
+   */
+  int getVertexId();
```

Comment:
Where is this being used?

Reply:
in hive's splitgenerator:
https://github.com/apache/hive/pull/5174/files#diff-e4b863c252f3407ad05ccb56548e9f8384e0af33ce4bf6841a0ce11b173bf00dR197